test/lib/linter/rules/snapshots/NoDeprecatedApi.ts.md

```diff
@@ -54,15 +54,130 @@ Generated by [AVA](https://avajs.dev).
             severity: 2,
           },
           {
-            column: 24,
+            column: 3,
             line: 16,
+            message: 'Bootstrap parameter \'data-sap-ui-animation\' should be replaced with data-sap-ui-animation-mode',
+            messageDetails: 'Migrate to \'data-sap-ui-animation-mode\' attribute AnimationMode (https://ui5.sap.com/1.120/#/api/module:sap/ui/core/AnimationMode)',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 3,
+            line: 17,
+            message: 'Abandoned bootstrap parameter \'data-sap-ui-areas\' should be removed',
```

Comment:
This message is missing some details. It's not clear why it can be removed. AFAIK uiareas are created automatically when using `placeAt` for a control, but I think this should be explained to the user.